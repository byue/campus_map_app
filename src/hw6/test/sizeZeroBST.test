# Simple test of breadth-first search algorithm.
# Verifies output of BFS for size 0 graph.

CreateGraph characters

FindPath characters GORILLA-MAN VENUS-II