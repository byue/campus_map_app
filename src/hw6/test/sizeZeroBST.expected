# Simple test of breadth-first search algorithm.
# Verifies output of BFS for size 0 graph.

created graph characters

unknown character GORILLA-MAN
unknown character VENUS-II